lib/views/pr-detail-view.js

```diff
@@ -345,10 +349,14 @@ export class BarePullRequestDetailView extends React.Component {
     this.props.relay.refetch({
       repoId: this.props.repository.id,
       issueishId: this.props.pullRequest.id,
-      timelineCount: 100,
```

Comment:
I noticed when testing that this changes the appearance of the timeline on the overview tab a bit, since we're only fetching half the data we used to and users have to click "load more" more often.  Not the biggest deal, just something to be aware of / make sure we're ok with before changing this const.